nbgrader/apps/fetchfeedbackapp.py

```diff
@@ -26,25 +26,22 @@ class FetchFeedbackApp(NbGrader):
     flags = flags
 
     examples = """
-        Fetch feedback for an assignment that an instructor has released. MORE INFO NEEDED
+        Fetch feedback for an assignment, if an instructor has released it.
         For the usage of students.
 
-        You can run this command from any directory, but usually, you will have a
-        directory where you are keeping your course assignments.
+        You can run this command from any directory, but usually, you will run
+        it from the directory where you are keeping your course assignments.
 
-        To fetch an assignment by name into the current directory:
+        To fetch feedback, you must first know the `course_id` for your course.
```

Comment:
If the course_id is set in the `nbgrader_config.py` file, then you don't need to explicitly pass it as a flag, so I would maybe leave the current documentation that shows it without the flag.